src/file_layer.ts

```diff
@@ -101,7 +119,20 @@ export class FileLayer extends AbstractEmsService {
     return this._emsClient.getFileApiUrl();
   }
 
-  private _getDefaultFormat(): EmsFileLayerFormatGeoJson | EmsFileLayerFormatTopoJson {
+  private _getFormatUrl(format: EMSFormats) {
+    const url = this._proxyPath + this._getAbsoluteUrl(format.url);
+    return this._emsClient.extendUrlWithParams(url);
+  }
+
+  private _getFormatMeta(format: EMSFormats) {
+    if ('meta' in format) {
+      return format.meta;
+    } else {
+      return;
+    }
```

Comment:
Would it work simply with the optional chaining?

```suggestion
    return format?.meta
```

Reply:
I think optional chaining in your suggestion only tests if `format` is defined. In our case, we know `format` is defined, but we don't know if the `meta` property of `format` is defined.